dataactbroker/fileRoutes.py

```diff
@@ -136,6 +136,11 @@ def complete_generation(generation_id):
         file_manager = FileHandler(request, is_local=is_local, server_path=server_path)
         return file_manager.complete_generation(generation_id)
 
+    @app.route("/v1/upload_detached_file/", methods=["POST"])
+    def upload_detached_file():
```

Comment:
I think it's worthwhile to follow the standard put in place since it can used simply for assessing which fields are required and what their type should be. We probably don't need the `validate` part though